README.md

```diff
@@ -67,35 +67,73 @@ Note that the specification files primarily define the structure (syntax) of a p
-       schemas.
-     * If the change is in semantic rules, add a constraint in the rule, so they only apply on
-       the indicated version range.
-     * Add a changelog entry in the `spec/changelog.yml` file in the section of this major.
-  * If the proposed change makes the specification looser than it is at `x.y.z`, the new version number will be `x.Y.0`, where `Y = y + 1`. That is, we bump up the minor version and create a new changelog section in the `spec/changelog.yml` file. Note that adding new, but optional, constraints to a specification is a change that makes a specification looser.
-  * If the proposed change does not change the strictness of the specification at `x.y.z`, the new version number will be `x.y.Z`, where `Z = z + 1`. That is, we bump the patch version.
-     * Apply the proposed changes to the existing specification under the `spec` folder.
-     * Set the root-level `version` property in the specification's root `spec.yml` file to `x.y.Z`.
-     * Add a changelog entry in the `spec/changelog.yml` file in the section for
-       this version.
+Consider a proposal to change the specification in some way. The version number
+of the changed specification must be determined as follows:
+
+  * If the proposed change introduces changes in the format of the files that
+    require manual changes in packages, the new version number will be `X.0.0`,
+    where `X = x + 1`. That is, we bump up the major version.
+    * Exception: When the proposed change is intended to address existing issues
+      in packages like ambiguous mappings or security risks.
+    * Exception: When the proposed change affects a feature marked as technical
+      preview.
+  * If the proposed change introduces support for a new feature that requires
+    explicit support in the Stack, the new version will be `x.Y.0`, where
+    `Y = y + 1`. That is, we bump up the minor version. See note below about
+    compatibility between packages and the Stack.
+  * Any other change would be included in the next patch version, `x.y.Z` where
+    `Z = z + 1`. This includes any change on validation that doesn't neccesarily
+    lead to a change in the behaviour of the installed package.
+
+If the change is in a schema file, add a JSON patch in the `versions` section to
+continue supporting the previous format.
+
+If the change is in semantic rules, add a constraint in the rule, so they only
+apply on the indicated version range and package types.
+
+Remember to add a changelog entry in `spec/changelog.yml` for any change in the
+spec. If no section exist for the version determined by the above rules, please
+add the new section. Multiple `next` versions may exist at the same moment if
+multiple versions are in development.
 
 ## Version Compatibility between Packages and Specifications
 
 A package specifying its `format_version` as `x.y.z` must be valid against specifications in the semantic version range `[x.y.z, X.0.0)`, where `X = x + 1`.
```

Comment:
Just double checking about this section, this consideration is still valid about the ranges of valid specifications for a given package, no ? 

Would a package using 3.0.1 as format_version be still valid checking with package-spec 3.1.0 o 3.2.0 versions ? And it will be invalid for 4.0.0. Is that still right?

According to this item I would say there is no need to update this section:
> If the proposed change introduces support for a new feature that requires
    explicit support in the Stack, the new version will be `x.Y.0`, where
    `Y = y + 1`. That is, we bump up the minor version. See note below about
    compatibility between packages and the Stack.

Reply:
>Would a package using 3.0.1 as format_version be still valid checking with package-spec 3.1.0 o 3.2.0 versions ? And it will be invalid for 4.0.0. Is that still right?

Yes, in principle it should be compatible with any 3.x version, but doesn't need to be compatible with 4.x. This is not completely true, neither has been, because we have introduced incompatible changes for bugfixes and so on (the exceptions mentioned above).

Reply:
Ok, agreed!